tests/providers/airbyte/hooks/test_airbyte.py

```diff
@@ -128,7 +128,7 @@ def test_wait_for_job_cancelled(self, mock_get_job):
 
     @requests_mock.mock()
     def test_connection_success(self, m):
-        m.get(self.health_endpoint, status_code=200,)
+        m.get(self.health_endpoint, status_code=200)
```

Comment:
This change is not related :)
We also had a broken CI so another rebase so fix the failing tests